gax/src/main/java/com/google/api/gax/rpc/UnaryCallSettings.java

```diff
@@ -130,9 +153,35 @@ protected Builder(UnaryCallSettings<RequestT, ResponseT> unaryCallSettings) {
       return this;
     }
 
+    /**
+     * Returns the underlying {@link RetrySettings.Builder}, which allows callers to augment the
+     * existing {@link RetrySettings}.
+     */
+    public RetrySettings.Builder retrySettings() {
```

Comment:
This puts us in situation when we have `getRetrySettgings()`, which returns settings and `retrySettings()` which returns the builder. It is confusing. Maybe explicitly call it `getRetrySettingsBuilder()`? Yes, it is longer and uglier, but at least makes it clear what it is.

Reply:
I'd like to keep it consistent with {Unary,ServerStreaming}CallSettings. We don't suffix the methods with Builder:

```java
// Current CallSettings 
MyStubSettings.newBuilder()
   .myMethod()
   .setBlah()

// Not
MyStubSettings.newBuilder()
   .myMethodSettingsBuilder()
   .setBlah()
```

Reply:
Ok, consistency is important